### PR TITLE
Drop mom into week of preg seq

### PR DESCRIPTION
**Pull request by gsvr**
Currently we drop mothers into message seq_id 1 for protocol. It needs to use gestation calc to drop in mid-stream.

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/chw.js [hunk 1]
```diff
@@ -553,6 +553,7 @@ go.app = function() {
                         opts = go.utils.subscription_type_and_rate(self.contact, self.im);
                         self.contact.extra.subscription_type = opts.sub_type.toString();
                         self.contact.extra.subscription_rate = opts.sub_rate.toString();
+                        self.contact.extra.subscription_seq_start = opts.sub_seq_start.toString();
```
Comment (gsvr) [inline src/chw.js:556]: This may be unnecessary.

Comment (imsickofmaps) [inline src/chw.js:556]: No, still worth doing.

**Comment by imsickofmaps** [comment]
:+1: good work

**Pull request merged**